app/models/post.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Post < ApplicationRecord
+  validates :title, uniqueness: true, presence: true
+  validates :body, presence: true
+  validates :author_id, presence: true
```

Comment:
- [ ] `belongs_to :author` already adds presence validation to the author. So we don't need to validate the presence of `author_id`.

Reply:
fix